code/modules/clothing/rogueclothes/rings.dm

```diff
@@ -18,11 +18,54 @@
 	icon_state = "ring_s"
 	sellprice = 33
 
+/obj/item/clothing/neck/roguetown/psicross/ring   /////////////////////// cast focus ring for acolytes and shit.
```

Comment:
Why is this a neck item if its a ring?

Reply:
Why not... uh.. that seems like it should be fixed if thats the case on other rings and not piled onto, makes it extremely confusing